app/controllers/api/logs_controller.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+class Api::LogsController < ApiController
+  self.permitted_attrs = :encryptable_id
+
+  def index(options = {})
+    authorize(team, :index?, policy_class: LogPolicy)
+    render({ json: fetch_entries,
+             each_serializer: list_serializer,
+             root: 'Logs_'.pluralize }
+             .merge(render_options)
+             .merge(options.fetch(:render_options, {})))
+  end
+
+  def fetch_entries
```

Comment:
I am putting this comment here as this seems like the place where you would solve this problem.
So as we create an Event every time someone opens a encryptable we will (obviously) create **a lot** of events (to test this i created a few thousand events by hand).
It however looks like you are not paginating the results, in turn, loading these thousands events every time someone opens an encryptable.
This will obviously hinder performance and is therefore not advisable. 
The issue states: 
``` 
Find a solution for only showing some of the present log entries if there's a lot
- pagination
- or infinite scroll
- or only display last 20 entries (sql limit 20)
```
Please implement that as well. (a simple solution as limiting the query here to the `.last(20)` or similar should do)

Reply:
I created a simplified "load more" button which loads 10 logs at a time. To avoid having to get the total amount of logs from the backend most of the logic is implemented in the frontend. So, the button is shown whenever the amount of requested logs from the backend is smaller than or equal to the number of logs displayed. This means that when the number of logs is divisible by 10 the button can be pressed one more time (without anything happening) before it disappears. If that solution doesn't work for you we can also just use the previous commit where simply 20 logs are loaded, without an option to see more.

Reply:
That seems like a really nice solution for now, thanks! :car: :dash: